doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1676,68 +1615,100 @@ Controlling the proof flow
    .. exn:: @ident is used in conclusion.
       :undocumented:
 
-.. tacn:: generalize @term
-   :name: generalize
-
-   This tactic applies to any goal. It generalizes the conclusion with
-   respect to some term.
+   .. tacn:: specialize_eqs @ident
+      :undocumented:
 
-.. example::
+.. tacn:: generalize @one_term {? {+ @one_term } }
```

Comment:
Why not the following? (This is equivalent.)
```suggestion
.. tacn:: generalize {+ @one_term }
```

Reply:
Got it.  I notice this could also  be simplified.  I couldn't remember if this was intentional (perhaps to make grammar unambiguous between the 2nd and 3rd productions).

![image](https://user-images.githubusercontent.com/1253341/195169095-a9ae8fbe-2446-45cd-8502-4fa0776e6d73.png)

Reply:
Maybe it comes from how the grammar is defined in the `mlg`, and maybe indeed in this case we prefer to have a non-ambiguous grammar, but I think that for the doc, it would be better to simplify it like you suggest (especially as that won't really change the interpretation of the single `single_intropattern` case, which would be the same whatever form is chosen).